screens/AccountCloseConfirmForm.js

```diff
@@ -0,0 +1,452 @@
+                  	{t('cash')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Starting Cash 
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;{mostrecentShift.open.balance}</Text>
+                	</View>
+              	</View>
+
+              	{
+									mostrecentShift.close.closingShiftReport !== null && 
+									mostrecentShift.close.closingShiftReport.totalByPaymentMethod !== null && 
+									mostrecentShift.close.closingShiftReport.totalByPaymentMethod.hasOwnProperty('CASH') &&  (
+									<View style={styles.fieldContainer}>
+                		<View style={{flex: 3}}>
+                  		<Text style={[styles.fieldTitle]}>
+                  		Total Sales
+                  		</Text>
+                		</View>
+                		<View style={{flex: 1}}>
+                  		<Text>
+                  			$&nbsp;{                  				
+                  				mostrecentShift.close.closingShiftReport.totalByPaymentMethod.CASH.orderTotal
+												}
+                  		</Text>
+                			</View>
+              		</View>
+              	)}                        
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                  	Amount of payment
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;?</Text>
+                	</View>
+              	</View>     
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount of cash register should have
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
+              				name="name"
+              				component={InputText}
+              				//validate={isRequired}
+              				//placeholder={t('productName')}
+              				placeholder={"??"}
+              				secureTextEntry={false}
+              				height={35}
+            				/>
+                	</View>
+              	</View>
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount actually owned by cash register
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
```

Comment:
This shouldn't be text field. It should display the total cash entered from the previous screen.

Reply:
mostrecentShift.close.closingBalances.CASH.closingBalance